gcc/rust/ast/rust-ast-builder.cc

```diff
@@ -214,6 +214,13 @@ Builder::block (std::vector<std::unique_ptr<Stmt>> &&stmts,
 					       LoopLabel::error (), loc, loc));
 }
 
+std::unique_ptr<Expr>
+Builder::return_expr (std::unique_ptr<Expr> &&to_return)
+{
+  return std::unique_ptr<Expr> (
+    new ReturnExpr (std::move (to_return), {}, loc));
```

Comment:
Couldn't we use `make_unique here` ? I believe we can even get rid of our implementation since baseline cpp version has been bumped to cpp14.

Reply:
tried using it and it complains about not finding a matching constructor:

```c++
../../gcc/rust/ast/rust-ast-builder.cc:220:39: error: no matching function for call to ‘make_unique<Rust::AST::ReturnExpr>(std::remove_reference<std::unique_ptr<Rust::AST::Expr>&>::type, <brace-enclosed initializer list>, location_t&)’
  220 |   return std::make_unique<ReturnExpr> (std::move (to_return), {}, loc);
      |          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

but maybe I'm missing something